kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py

```diff
@@ -0,0 +1,26 @@
+from kentik_api.public.manual_mitigation import ManualMitigation
+import json
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass()
+class CreateRequest:
+    ipCidr: str
+    Comment: Optional[str]
```

Comment:
should be lowercase? API tester sends lowercase "comment" field

Reply:
ack